### PR TITLE
Create ycat.json

### PR DESCRIPTION
**Pull request by Chalk01**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://ycat.hstn.me/

### CODE REVIEW

**Review by Stef-DP — APPROVED** [review]

File: domains/ycat.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Chalk01",
+    "email": "c.superchayu@gmail.com"
+  }
```
Comment (Stef-00012) [inline domains/ycat.json:5]: Missing a comma here

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/ycat.json (hunk 1 above)
Comment (Chalk01) [reply domains/ycat.json:5]: Sorry, I'm new to github but what am I supposed to do now?

Comment (Stef-00012) [reply domains/ycat.json:5]: > Sorry, I'm new to github but what am I supposed to do now?

edit the file and add a `,` after the `}`

Comment (21Z) [reply domains/ycat.json:5]:
```suggestion
  },
```

Comment (Chalk01) [reply domains/ycat.json]: I've done it?

Comment (21Z) [reply domains/ycat.json]: wait.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**